dotcom-rendering/src/model/decideContainerPalette.ts

```diff
@@ -20,6 +21,6 @@ export const decideContainerPalette = (
 	if (palettes?.includes('BreakingPalette')) return 'BreakingPalette';
 	if (palettes?.includes('SpecialReportAltPalette'))
 		return 'SpecialReportAltPalette';
-	if (palettes?.includes('Branded')) return 'Branded';
+	if (palettes?.includes('Branded') && allCardsHaveBranding) return 'Branded';
```

Comment:
Even if a container has the 'Branded' label in the fronts tool, it will behave like a regular container if any of the cards are for actual (unpaid) Guardian content, effectively removing the Branded palette